### PR TITLE
#SRC-600: F license check

### PR DESCRIPTION
**Pull request by mikramarc**

### CODE REVIEW

File: ansible/docker_site.yml [hunk 1]
```diff
@@ -58,6 +58,7 @@
                                                       "python_code_coverage", "cpp_code_coverage"]}
     - {role: ci/code_style_check/python, when: run_build, tags: ["python_code_style_check", "code_style_check"]}
     - {role: ci/code_style_check/cpp, when: run_build, tags: ["cpp_code_style_check", "code_style_check"]}
+    - {role: ci/code_style_check/license, when: run_build, tags: ["check_license", "code_style_check"]}
```
Comment (AndriyPt) [inline ansible/docker_site.yml]: please remove "code_style_check" since it will fail in all existing projects :)

Comment (mikramarc) [reply ansible/docker_site.yml]: done in c1baab4

File: ansible/docker_site.yml [hunk 2]
```diff
@@ -58,6 +58,7 @@
                                                       "python_code_coverage", "cpp_code_coverage"]}
     - {role: ci/code_style_check/python, when: run_build, tags: ["python_code_style_check", "code_style_check"]}
     - {role: ci/code_style_check/cpp, when: run_build, tags: ["cpp_code_style_check", "code_style_check"]}
+    - {role: ci/code_style_check/licence, when: run_build, tags: ["check_licence"]}
```
Comment (AndriyPt) [inline ansible/docker_site.yml]: Maybe we should change it to American way of spelling license :)

File: ansible/roles/ci/doc/modules.md [hunk 3]
```diff
@@ -10,7 +10,7 @@
   * **all_tests** - run **software_tests** tests described above as well as hardware tests
   * **benchmarking** - run benchmarking of the functionality based on rostest
   * **check_build** - long running module. Compile independently each package in the project cleaning devel and build folder beforehand and check if all dependencies are declared in CMakeList.txt correctly.
-  * **code_style_check** - run [roslint](http://wiki.ros.org/roslint) to check C++ and Python code in all packages of the project.
+  * **code_style_check** - run [roslint](http://wiki.ros.org/roslint) to check C++ and Python code in all packages of the project. Also, check if copyright notice is present in all the files and the license file exsists in the repository.
```
Comment (AndriyPt) [inline ansible/roles/ci/doc/modules.md]: Documentation should contain new role *check_licence* modules description

File: ansible/roles/ci/doc/modules.md [hunk 4]
```diff
@@ -11,6 +11,7 @@
   * **benchmarking** - run benchmarking of the functionality based on rostest
   * **check_build** - long running module. Compile independently each package in the project cleaning devel and build folder beforehand and check if all dependencies are declared in CMakeList.txt correctly.
   * **code_style_check** - run [roslint](http://wiki.ros.org/roslint) to check C++ and Python code in all packages of the project.
+  * **check_license** - check if copyright notice is present in all the files and the license file exsists in the repository.
```
Comment (AndriyPt) [inline ansible/roles/ci/doc/modules.md]: "and the license file exsists" should be "and the LICENSE file exsists"

**Pull request merged**